controllers/mongodb_status_options.go

```diff
@@ -60,6 +60,27 @@ func (m mongoUriOption) GetResult() (reconcile.Result, error) {
 	return result.OK()
 }
 
+func (o *optionBuilder) withVersion(mdb *mdbv1.MongoDBCommunity) *optionBuilder {
```

Comment:
[q] why are we passing `mdb` instead of simply the version string?

Reply:
good point, I'll change it to passing the string